.gitignore

```diff
@@ -12,6 +12,9 @@ yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
 
+# Configuration file
+config.json*
```

Comment:
This should just match `config.json`. The current selector will also grab things like a future `config.json.template` we may add.